src/tools/cephfs/run-tox.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I really wish the cephfs-shell would be a part of a proper python package. We're again reinventing the wheel all over again. We already have multiple run-tox scripts in the repo. :-1: for adding yet another one. 

In #28915 I'm trying to come up with a base for a proper structure for adding python sub-packages.

Reply:
I'm okay with doing with that but it shouldn't block this PR even if we may undo some of it later.